sign/sign.go

```diff
@@ -718,7 +724,24 @@ func (s *Signer) IsFileSigned(ctx context.Context, path string) (bool, error) {
 		return false, err
 	}
 
-	uuids, err := s.impl.FindTLogEntriesByPayload(ctx, rClient, blobBytes)
+	var b64sig string
+	if isb64(blobBytes) {
+		b64sig = string(blobBytes)
+	} else {
+		b64sig = base64.StdEncoding.EncodeToString(blobBytes)
+	}
+
+	pubKey, err := sigs.PublicKeyFromKeyRef(ctx, ko.KeyRef)
+	if err != nil {
+		return false, err
+	}
+
+	pubBytes, err := sigs.PublicKeyPem(pubKey, signatureoptions.WithContext(ctx))
+	if err != nil {
+		return false, err
+	}
```

Comment:
nit: I would prefer to have these errors wrapped as we're calling out to external libraries and the context may be different.

Reply:
done